Core/clim-basic/clim-basic.asd

```diff
@@ -24,7 +24,9 @@
                  (:file "region-bounding-rectangles")
                  (:file "region-transformations")
                  (:file "region-predicates")
-                 (:file "region-composition")))
+                 (:file "region-composition")
```

Comment:
The commit message says

> add missing "general" methods for region-difference (clearly missing)

Please list the methods in the commit message since it is really hard to tell from the diff.

Reply:
fixed the commit message with a brief explanation why it is "clearly" missing.